src/ringct/rctOps.cpp

```diff
@@ -350,6 +351,7 @@ namespace rct {
     //This takes the outputs and commitments
     //and hashes them into a 32 byte sized key
     key cn_fast_hash(const ctkeyV &PC) {
+        if (PC.empty()) return rct::hash2rct(crypto::cn_fast_hash("", 0));
```

Comment:
Are there ever expected cases where PC is really empty? Or, should these be asserts? I'm not sure if this is a bugfix or defensive.

Reply:
I doubt there are cases where it can be, but I wouldn't bet there won't ever.